CHANGELOG.md

```diff
@@ -26,8 +26,8 @@ Non-backwards compatible changes
   `boolFilter` and `boolPartition`, and have been replaced by new functions
   `filter` and `partition` which use decidable predicates instead of boolean-valued
   functions.  The former encouraged poor programming style in a dependantly-typed 
-  language such as Agda. Proofs for `boolFilter` and `boolPartition` have also been
-  renamed accordingly.
+  language such as Agda. Proofs for `filter` and `partition` have also been
+  updated and Prenamed accordingly.
```

Comment:
Is `Prenamed` a typo or some cunning use of language?

Reply:
Definitely typo! Will fix.